src/libraries/System.Runtime/tests/System/Reflection/ReflectionCacheTests.cs

```diff
@@ -60,5 +118,13 @@ private static Action<Type[]> GetClearCacheMethod()
             Assert.NotNull(clearCache);
             return clearCache.CreateDelegate<Action<Type[]>>();
         }
+
+        private static bool HotReloadDeltaApplied()
+        {
+            Type updateHandler = typeof(Type).Assembly.GetType("System.Reflection.Metadata.RuntimeTypeMetadataUpdateHandler", throwOnError: true, ignoreCase: false);
+            PropertyInfo hotReloadApplied = updateHandler.GetProperty("HotReloadDeltaApplied", BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Static);
```

Comment:
Do we really need to be testing this internal implementation detail?

Reply:
not necessarily